include/triton/Dialect/TritonGPU/IR/TritonGPUAttrDefs.td

```diff
@@ -554,6 +558,7 @@ The data will be distributed between threads as follows:
   let parameters = (
     ins
     "unsigned":$nonKDim,
+    "unsigned":$kDim,
```

Comment:
@zhanglx13
Looks like we need this `kDim`/`k_base` field after all, because we have two similar instructions for BF16 x BF16 -> FP32 operation with MFMA 32 family of instructions.

The difference between them in number of elements per thread.
MI100 supports only one of these instructions, MI200 supports both.

In ideal world I would like to add this attribute to `dotOperand` encoding (near this mmaV2 width attribute), but this requires some changes in common code, which I don't want to make, because it will complicate IFU process.

Reply:
Yeah. If you don't want to reuse the `MMAv2kWidth` field and add a new attribute to dotOperand, adding it to the mfmaEncoding seems to be the only option we have.
Or, we can persuade upstream to rename `MMAv2kWidth` to something like `kWidth`, which should not be too much to ask.
But before that, I'm ok to add it to the mfmaEncoding.

Reply:
> Or, we can persuade upstream to rename MMAv2kWidth to something like kWidth, which should not be too much to ask.

This is a great idea. Hope upstream will agree on this :)

Reply:
Upstream (Phil) agreed to rename MMAv2kWidth to kWidth. You can proceed when https://github.com/ROCmSoftwarePlatform/triton/pull/299 is merged.

Reply:
#299 is merged